.asf.yaml

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+
+github:
+  description: "Web-based notebook that enables data-driven, interactive data analytics and collaborative documents with SQL, Scala and more."
+  homepage: https://zeppelin.apache.org/
+  labels:
+    - scala
+    - java
+    - big-data
+    - zeppelin
+    - javascript
```

Comment:
I just copied the current labels on the repository here. But feel like they need to be improved. Any suggestions?

Reply:
I would add `spark`, `flink`, `database`, `nosql` - the first two are very popular.

Reply:
Added them on the list. Thanks for the suggestion.

I'll merge this to master, while it can be updated anytime via pull request.